### PR TITLE
FT21-2

### PR DESCRIPTION
**Pull request by amilan17**
https://github.com/wmo-im/BUFR4/issues?q=is%3Aopen+is%3Aissue+milestone%3AFT-2021-2

### CODE REVIEW

File: BUFRCREX_CodeFlag_en_08.csv [hunk 1]
```diff
@@ -439,17 +439,21 @@ FXY,ElementName_en,CodeFigure,EntryName_en,EntryName_sub1_en,EntryName_sub2_en,N
...
+008043,Atmospheric chemical or physical constituent type,10-24,Reserved,,,,Operational
 008043,Atmospheric chemical or physical constituent type,25,Particulate matter < 1.0 microns,,,,Operational
 008043,Atmospheric chemical or physical constituent type,26,Particulate matter < 2.5 microns,,,,Operational
 008043,Atmospheric chemical or physical constituent type,27,Particulate matter < 10 microns,,,,Operational
 008043,Atmospheric chemical or physical constituent type,28,Aerosols (generic),,,,Operational
 008043,Atmospheric chemical or physical constituent type,29,Smoke (generic),,,,Operational
 008043,Atmospheric chemical or physical constituent type,30,Crustal material (generic dust),,,,Operational
 008043,Atmospheric chemical or physical constituent type,31,Volcanic ash,,,,Operational
-008043,Atmospheric chemical or physical constituent type,32-200,Reserved,,,,Operational
+008043,Atmospheric chemical or physical constituent type,32,Cloud,,,,Operational
```
Comment (chenxiaoxia2019) [inline BUFRCREX_CodeFlag_en_08.csv:451]: - [ ] checked

File: BUFR_TableD_en_09.csv [hunk 2]
```diff
@@ -216,6 +216,35 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
...
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,301132,Common header sequence,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,008043,Atmospheric chemical or physical constituent type,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,025061,Software Version Number,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,302004,General cloud information,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,302005,Cloud layer,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,008092,Measurement uncertainty expression,,(18) The Measurement uncertainty expression and significance descriptors 0 08 092 and 0 08 093 relate to all parameter uncertainties provided in the backscatter data section.,Operational
```
Comment (chenxiaoxia2019) [inline BUFR_TableD_en_09.csv:224]: @amilan17 I suggest here that (18) be replaced by (see Note 18)

File: BUFR_TableD_en_09.csv [hunk 3]
```diff
@@ -216,6 +216,35 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
...
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015068,Uncertainty in particle extinction coefficient,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015076,Particle lidar ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015077,Uncertainty in lidar ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015078,Particle depolarization ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015072,Uncertainty in depolarization ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,033002,Quality information,,(19) The Quality information descriptor 0 33 002 relates to all parameters provided in the Backscatter data section and is given for each respective vertical bin of the profile measurement.,Operational
```
Comment (chenxiaoxia2019) [inline BUFR_TableD_en_09.csv:245]: @amilan17 Here "(19)" to be replaced by "(see Note 19)"

File: BUFRCREX_CodeFlag_en_08.csv [hunk 4]
```diff
@@ -439,17 +439,21 @@ FXY,ElementName_en,CodeFigure,EntryName_en,EntryName_sub1_en,EntryName_sub2_en,N
...
+008043,Atmospheric chemical or physical constituent type,33,Cloud and aerosols,,,,Operational
+008043,Atmospheric chemical or physical constituent type,34-200,Reserved,,,,Operational
 008043,Atmospheric chemical or physical constituent type,201-254,Reserved for local use,,,,Operational
 008043,Atmospheric chemical or physical constituent type,255,Missing value,,,,Operational
+008046,Atmospheric chemical or physical constituent type,,,,,"(See Common Code table C–14 in Part C/c.)",Operational
```
Comment (chenxiaoxia2019) [inline BUFRCREX_CodeFlag_en_08.csv:456]: - [ ] checked

**Review by chenxiaoxia2019 — COMMENTED** [review]
@amilan17 All files are reviewed. 1) Issue 55 is not yet merged.  2) Issue 73: I updated the proposal in GitHub by adding "in Part C/c."

File: BUFR_TableD_en_09.csv (hunk 2 above)
Comment (amilan17) [reply BUFR_TableD_en_09.csv:224]: done

File: BUFR_TableD_en_09.csv (hunk 3 above)
Comment (amilan17) [reply BUFR_TableD_en_09.csv]: done

**Review by amilan17 — COMMENTED** [review]
all are ready for FT

**Pull request merged**